src/com/projectkorra/projectkorra/ProjectKorra.java

```diff
@@ -30,13 +17,18 @@
 import com.projectkorra.projectkorra.hooks.PlaceholderAPIHook;
 import com.projectkorra.projectkorra.hooks.WorldGuardFlag;
 import com.projectkorra.projectkorra.object.Preset;
+import com.projectkorra.projectkorra.region.RegionProtection;
 import com.projectkorra.projectkorra.storage.DBConnection;
-import com.projectkorra.projectkorra.util.Metrics;
-import com.projectkorra.projectkorra.util.RevertChecker;
-import com.projectkorra.projectkorra.util.StatisticsManager;
-import com.projectkorra.projectkorra.util.TempBlock;
-import com.projectkorra.projectkorra.util.Updater;
+import com.projectkorra.projectkorra.util.*;

```

Comment:
Seems that IntelliJ is auto sorting the imports. If you are removing unused imports, make sure it won't condense them like this